L72-core-google_default_credentials-extension.md

```diff
@@ -0,0 +1,65 @@
+Allow Call Credentials to be Specified in `grpc_google_default_credentials_create`
+----
+* Author(s): rbellevi
+* Approver: markdroth
+* Status: Draft
+* Implemented in: Core
+* Last updated: July 8th, 2020
+* Discussion at: https://groups.google.com/g/grpc-io/c/fZNm4pU8e3s/m/2Be8u1n7BQAJ
+
+## Abstract
+
+This document proposes that `grpc_google_default_credentials_create` be
+amended to allow the user to specify their desired call credentials.
+
+## Background
+
+The Google default credentials created by the
+`grpc_google_default_credentials_create` function in Core enable connection to
+Google services via a combination of ALTS and SSL credentials, along with a special oauth2
+token that must assert the same identity as the channel-level ALTS credential.
```

Comment:
nit: related to below comment, for accuracy lets replace "must assert" with "by default asserts"

Reply:
Maybe describe how the token that google default credentials uses is decided per the Application Default Credentials spec

Reply:
these comments are still relevant, to improve accuracy of the wording here

Reply:
I missed the `s/must/by default/`. Sorry about that. I added a numbered list describing how ADC works in detail.